### ISSUE TITLE
batch normalization

### ISSUE THREAD

**Issue by yzbx** [ISCAS007/torchseg#13]
- [batch normalization implementation discuss](https://github.com/PaddlePaddle/Paddle/issues/3684)
- [batch normalization survey](http://www.deeplearningpatterns.com/doku.php?id=batch_normalization)
